mne/stats/regression.py

```diff
@@ -134,3 +143,208 @@ def _fit_lm(data, design_matrix, names):
+        overload your computer. Typically, 100 hz sampling rate is
+        appropriate - or using the decim keyword (see below).
+    events : array
+        An n x 3 array, where the first column corresponds to samples in raw
+        and the last to integer codes in event_id.
+    event_id : dict
+        As in Epochs; a dictionary where the values may be integers or
+        list-like collections of integers, corresponding to the 3rd column of
+        events, and the keys are condition names.
+    tmin : float | dict
+        If float, gives the lower limit (in seconds) for the time window for
+        which all event types' effects are estimated. If a dict, can be used to
+        specify time windows for specific event types: keys correspond to keys
+        in event_id and/or covariates; for missing values, the default (-.1) is
+        used.
+    tmax : float | dict
+        If float, gives the upper limit (in seconds) for the time window for
+        which all event types' effects are estimated. If a dict, can be used to
+        specify time windows for specific event types: keys correspond to keys
+        in event_id and/or covariates; for missing values, the default (1.) is
+        used.
+    covariates : dict-like | None
+        If dict-like, values have to be array-like and of the same length as
+        the columns in ```events```. Keys correspond to additional event
+        types/conditions to be estimated and are matched with the time points
+        given by the first column of ```events```. If None, only binary events
+        (from event_id) are used.
+    reject : bool | dict
+        Activate rejection parameters based on peak-to-peak amplitude in
+        continuously selected subwindows. If reject is None, no rejection is
+        done. If True, the following default is employed:
+            reject = dict(grad=4000e-12, # T / m (gradiometers)
+                          mag=4e-11, # T (magnetometers)
+                          eeg=40e-5, # uV (EEG channels)
+                          eog=250e-5 # uV (EOG channels)
+                         )
+        If dict, keys are types ('grad' | 'mag' | 'eeg' | 'eog' | 'ecg') and
+        values are the maximal peak-to-peak values to select rejected epochs.
+    tstep : float
+        Length of windows for peak-to-peak detection.
+    decim : int
+        Decimate by choosing only a subsample of data points. Highly
+        recommended for data recorded at high sampling frequencies, as
+        otherwise huge intermediate matrices have to be created and inverted.
+    picks : None | list
+        List of indices of channels to be included. If None, defaults to all
+        MEG and EEG channels.
+    solver : str | function
+        Either a function which takes as its inputs the predictor matrix X
+        and the observation matrix Y, and returns the coefficient matrix b;
+        or a string (for now, only 'defaults'), in which case the solver used
```

Comment:
s/defaults/default

also maybe instead of 'default` which is ambiguous, it should just be something like `pseudoinverse` or `pinv`, or some more standard name for this simple technique?